packages/warriorjs-cli/src/Profile.js

```diff
@@ -1,5 +1,7 @@
 import fs from 'fs';
 import path from 'path';
+import pathType from 'path-type';
```

Comment:
@glneto This import should be in the second group of imports, below `makeDir`, because it's a third-party library.